src/rules/no-nodejs-modules.js

```diff
@@ -7,6 +9,23 @@ function reportIfMissing(context, node, allowed, name) {
   }
 }
 
+function isIgnored(ignored, filename) {
+  return ignored.some(c => (
+    minimatch(filename, c) ||
+    minimatch(filename, path.join(process.cwd(), c))
+  ))
+}
+
+function guaranteeArray(item) {
+  if (item instanceof Array) {
```

Comment:
`intsanceof` is never reliable; `Array.isArray(item)` is the proper way to check.

However, this entire function can be replaced with `[].concat(item || [])`, i believe